src/main/java/com/google/devtools/build/lib/analysis/test/AnalysisTestActionBuilder.java

```diff
@@ -37,25 +37,24 @@ private AnalysisTestActionBuilder() {}
    */
   public static void writeAnalysisTestAction(
       RuleContext ruleContext, AnalysisTestResultInfo testResultInfo) {
-    // TODO(laszlocsomor): Use the execution platform, not the host platform.
-    boolean isExecutedOnWindows = OS.getCurrent() == OS.WINDOWS;
+    boolean isTargetOsWindows = ruleContext.isTargetOsWindows();
```

Comment:
cc @brandjon @tetromino 
No need to do anything (maybe a TODO) otherwise just a driveby comment. 

Do you know why we need special casing`AnalysisTestResultInfo` for Windows? This should also be general like `RuleContext`.